cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -523,3 +439,18 @@ func (oi *OVFImporter) CleanUp() {
 		}
 	}
 }
+
+func (oi *OVFImporter) importWithModule(settings daisyutils.TranslationSettings) error {
```

Comment:
since the context is import, not translation, `translationSettings` is more descriptive instead of plain `settings`

Reply:
updated to take the `osID`, since that's the only portion that's required.